### DISCUSSION TITLE
Daily 8/11

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#280] [General]
Ayer armé unos diagramas de caso de uso para el CRUD y para subastas, por ahí arme un par más como para tener para elegir, aunque son todos iguales. También grabé mi video para el final. Hoy voy a ver si puedo editar algunos para ganar unos segundos para el resto. Ayer también hice el final de inglés así que en teoría promocioné. Hoy probablemente haga merge a main y deploye Bidón en Github Pages, también voy a revisar si falta algo más al sitio de Almoneda. También quiero mover las carpetas como para que quede la estructura final de la entrega.

**Comment by AhumadaCeleste** [comment]
Ayer, envié el vídeo para el proyecto, estudie Inglés a la tarde y me fui a dormir con la clase de full en vez de escuchar mi asmr relajante. Hoy Inglés. Mañana Inglés. Pasado, Ingles y pasado, pasado... inglés... y Phyton

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve estudiando Inglés y agregando algunos enlaces a los pie de páginas.
Hoy estaré estudiando Inglés, también voy a tratar de subir todas las modificaciónes al repositorio.

**Comment by Brian-Almada** [comment]
Ayer cree el video para el proyecto y estudié ingles. Hoy estuve estudiando ingles nuevamente.
